src/update-feeds/update-feeds.ts

```diff
@@ -0,0 +1,91 @@
+import { go } from '@api3/promise-utils';
+import { range, size } from 'lodash';
+
+import type { Chain } from '../config/schema';
+import { logger } from '../logger';
+import { getState } from '../state';
+import { isFulfilled, sleep } from '../utils';
+
+import { type ActiveDapisBatch, getStaticActiveDapis } from './temporary-contract-mock';
+
+export const startUpdateFeedsLoops = async () => {
+  const state = getState();
+  const {
+    config: { chains },
+  } = state;
+
+  // Start update loops for each chain in parallel.
+  await Promise.all(
```

Comment:
We might want to make this a `Promise.allSettled` just in case something unexpected throws to not interrupt all loops in case it only affect one chain or provider? 🤔 

If you do that, Aaron implemented a [nice utility function to log errors](https://github.com/api3dao/operations-database/blob/main/src/workers/common-chain-utils.ts#L74)

Reply:
I purposefully used `Promise.all` because there should not be an error thrown.